README.md

```diff
@@ -7,3 +7,194 @@ Repository for generating pediatric template
 
 Instructions to download: https://intranet.neuro.polymtl.ca/computing-resources/data/git-datasets.html#download
 
+Same subjects kept for both templates (N = 18). Some notes below on why some subjects are missing:
+* Incomplete data acquisition for `sub-107`, `sub-110` and `sub-125`: subjects were only kept if they had re-composed T1w and T2w images (images are generated by "stitching" the top and bottom acquisitions with each other).
+* `sub-108`: removed because disc 3 (above vertebrum 3) is blurry in straightened T2w image.
+* `sub-123`: removed because disc 10 (above vertebrum T3) is blurry in straightened T1w image.
+
+> for `sub-103` and `sub-109` T1w images, the spinal cord masks were not great. I did not need to manually fix the SC mask labels for these two subjects because their centerline files were automatically generated and used by the [template pipeline](https://github.com/neuropoly/template).
+
+
+## Regenerating T1w and T2w Templates
+Step-by-step walk-through on how to generate T1w and T2w pediatric templates from philadelphia-pediatric dataset.
+All code below should be run in your command-line (bash) on CCDB. Each step begins from your **working directory**: `/PATH/TO/pediatric-template`.
+
+### 1. Get data from git annex
+```
+git clone git@data.neuro.polymtl.ca:datasets/philadelphia-pediatric
+cd philadelphia-pediatric
+git checkout -b nb/add-labels
```

Comment:
This branch is no longer required. Everything has been merged into the master branch. See -- https://github.com/neuropoly/data-management/issues/248#issuecomment-1745287547